master/docs/developer/master-worker.rst

```diff
@@ -39,7 +39,8 @@ Bot methods
 The worker-side Bot object has the following remote methods:
 
 :meth:`~buildbot_worker.pb.BotPb.remote_getCommands`
-    Returns a list of ``(name, version)`` for all commands the worker recognizes.
+    Returns a dictionary ``Dict[name, version]`` for all commands the worker recognizes.
```

Comment:
I'm not sure what this notation means -- are names the keys and versions the values?